infra/discovery/src/apollo/resolvers.js

```diff
@@ -46,7 +72,107 @@ const resolvers = {
         nodeEnv: process.env.NODE_ENV ? process.env.NODE_ENV : 'undefined'
       }
     }
+  },
+  Mutation: {
+    async listingSetScoreTags(_, data, context) {
+      const auth = await authenticate(context.discoveryAuthToken)
+      if (!auth) {
+        throw 'You are not logged in'
+      }
+      const { ethAddress } = auth
+
+      const { id, scoreTags } = data
+      await db.DiscoveryTagAction.create({
+        ethAddress,
```

Comment:
nit: we have decided on the convention to alway store eth addresses lowercases in the DB. update to:
`
ethAddress: ethAddress.toLowerCase()
`